snapcraft/commands/cleanbuild.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+snapcraft cleanbuild
+
+Uses a container using lxd to create the resulting snap.
+
+This is a way to guarantee that the snapcraft.yaml used with associated
+local sources is not using any dependencies local to the developer system.
+
+The cleanbuild command requires a properly setup lxd environment that
+can connect to external networks.
+
+Usage:
+  cleanbuild [options]
+
+Options:
+  -h --help             show this help message and exit.
+"""
+
+import logging
+import os.path
+import tarfile
+
+from docopt import docopt
+
+from snapcraft import repo
+from snapcraft.lxd import Cleanbuilder
+from snapcraft.common import format_snap_name
```

Comment:
hum, I dislike the import of symbols instead of modules.
But if you prefer it this way, as long as we are consistent I can live with it. Should we start using this style?